packages/react-router-dev/vite/plugin.ts

```diff
@@ -1092,21 +1093,22 @@ export const reactRouterVitePlugin: ReactRouterVitePlugin = () => {
               return;
             }
 
-            if (routeConfigChanged) {
-              logger.info(colors.green("Route config changed."), {
-                clear: true,
-                timestamp: true,
-              });
-            } else if (configCodeUpdated) {
-              logger.info(colors.green("Config updated."), {
-                clear: true,
-                timestamp: true,
-              });
-            }
+            let message = (() => {
+              if (configChanged) return "Config changed.";
+              if (routeConfigChanged) return "Route config changed.";
+              if (configCodeChanged) return "Config saved.";
+              if (routeConfigCodeChanged) return "Route config saved.";
+              return "Config saved.";
+            })();
```

Comment:
Couldn't help myself, had to see how this looked as a nested ternary

```suggestion
            // prettier-ignore
            let message =
              configChanged ? "Config changed." :
              routeConfigChanged ? "Route config changed." :
              configCodeChanged ? "Config saved." :
              routeConfigCodeChanged ? "Route config saved." :
              "Config saved.";
```

Feel free to leave as is if you prefer, but I do like the ternary form better / less ceremony